app/routes/analysis/analysis.py

```diff
@@ -2,13 +2,13 @@
 from typing import Any, Dict, List, Optional
 from uuid import UUID
 
-from fastapi import APIRouter, Depends, Path, Query
+from fastapi import APIRouter, Path, Query
 from fastapi.exceptions import HTTPException
 from fastapi.logger import logger
-from fastapi.openapi.models import APIKey
+# from fastapi.openapi.models import APIKey
```

Comment:
Do we want to disable API keys in staging again, or is that on accident?

Reply:
Yes, disabling them for now until Vizz devs are ready to test it.